src/components/shared-panel/shared-panel.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {
+  EmailShareButton,
+  FacebookMessengerShareButton,
+  FacebookShareButton,
+  TwitterShareButton,
+  WhatsappShareButton,
+} from 'react-share';
+import styled from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+import { IconEmail } from '../icons/social/email';
+import { IconFacebook } from '../icons/social/facebook';
+import { IconMessenger } from '../icons/social/messenger';
+import { IconTwitter } from '../icons/social/twitter';
+import { IconWhatsapp } from '../icons/social/whatsapp';
+
+export interface AuthorCredentialsProps {
+  facebookId: string;
+}
+
+const SharedPanelContainer = styled.div`
+  color: #668cff;
+  margin-top: 26px;
+  font-weight: bold;
+  font-size: 16px;
+`;
+
+const SharedPanelText = styled.p`
+  font-weight: bold;
+  font-size: 16px;
+`;
+
+const SocialLinks = styled.ul`
+  all: unset;
+  order: 2;
+
+  ${up('md')} {
+    order: 1;
+    margin-bottom: 20px;
+  }
+`;
+
+const SocialLinkItem = styled.li`
+  display: inline-block;
+  margin-right: 16px;
+  cursor: pointer;
+  &:last-of-type {
+    margin-right: 0;
+  }
+`;
+
+export const SharedPanel: React.FC<AuthorCredentialsProps> = ({
+  facebookId,
+}) => {
+  const shareUrl = typeof window !== `undefined` ? window.location.href : '';
+  const title = typeof window !== `undefined` ? window.document.title : '';
+
+  return (
+    <SharedPanelContainer>
+      <SharedPanelText>Artikel teilen</SharedPanelText>
+      <SocialLinks>
+        <SocialLinkItem>
+          <WhatsappShareButton url={shareUrl} title={title}>
+            <IconWhatsapp />
+          </WhatsappShareButton>
+        </SocialLinkItem>
+        <SocialLinkItem>
+          <FacebookMessengerShareButton appId={facebookId} url={shareUrl}>
```

Comment:
I would remove this. We probably won't create an app on Facebook Messanger, also the Facebook Share icon below can be used for Facebook stuff

Reply:
cool, thanks, because I was wondering if we want to let users contact the Author directly